webrtc-stats.html

```diff
@@ -268,6 +268,25 @@
           PeerConnection.
         </p>
       </section>
+      <section>
+        <h3>
+          Guidelines for getStats() results caching/throttling
+        </h3>
+        <p>
+          A call to getStats() touches many components of WebRTC and may take significant time to
+          execute. The implementation may or may not utilize caching or throttling of getStats()
+          calls for performance benefits, however any implementation must adhere to the following:
+        </p>
+        <p>
+          When the state of the PeerConnection visibly changes as a result of an API call, a
+          promise resolving or an event firing, subsequent getStats() calls MUST return up-to-date
```

Comment:
I think this is fine (and non-normative). I'm assuming this means "subsequent *new* getStats() calls". For in-flight calls, I don't think we have or want any guarantees.

Reply:
Indeed. Changed it to "subsequent new getStats() calls"